clock/rfc822_test.go

```diff
@@ -188,7 +188,7 @@ func TestParseRFC822Time(t *testing.T) {
 		{"Wed, November 03 2021 17:48:06 CST"},
 
 		// Timestamps without seconds.
-		{"Sun, 31 Oct 2021 12:10 -5000"},
```

Comment:
LOL. That was an interesting time zone 😄

Reply:
Curiously, this caused the test to fail in go1.23+.  They added validation on timezone offset out of range.